profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/plugins/tmgmt_poetry.ui.inc

```diff
@@ -341,13 +365,25 @@ class TMGMTPoetryTranslatorUIController extends TMGMTDefaultTranslatorUIControll
   }
 
   /**
-   * Helper module to check MT@EC settings.
+   * Helper module to check connector settings.
    */
   protected function checkPoetrySettings() {
     $messages = array();
     $poetry_settings = variable_get('poetry_service');
-    if (!isset($poetry_settings) || empty($poetry_settings)) {
-      $messages[] = t('Main "poetry_service" variable was not set or is empty. See the README.md file or read the wiki page.');
+
+    $translator = tmgmt_translator_load('poetry');
+    $settings = $translator->getSetting('settings');
+
+    if (empty($poetry_settings['address'])) {
+      $messages[] = t('The DGT webservice address is not set. Please contact your support team.');
+      $type = 'error';
+    }
+    if (empty($settings['callback_user']) || empty($settings['callback_password'])) {
+      $messages[] = t('The credentials for your drupal service are not correctly set. Please contact COMM EUROPA MANAGEMENT.');
```

Comment:
I thinks it's confusing to mention poetry here since this error is on drupal side. Besides DGT asked to not use the term 'poetry'. This message shows in the report in the line DGT connector so imo it's clear enough.
"..are not correctly set" is the right placing of words in english :)